src/utils/requests.js

```diff
@@ -1,14 +1,89 @@
 import axios from 'axios'
 
 export const userLogin = async (username, password) => {
-	try {
-		const response = await axios.post(
-			'http://localhost:5000/login',
-			{ userName: username, password: password },
-			{ headers: { 'Content-Type': 'application/json' } }
-		)
-		return response.data.user
-	} catch (err) {
-		return Promise.reject('User Not Found -> ' + err)
-	}
+    try {
+        const response = await axios.post(
+            'http://localhost:5000/login',
+            { userName: username, password: password },
+            { withCredentials: true }
+        )
+
+        return response.data.user
```

Comment:
Here you have to check if the response has the proper format. You could return undefined and the components that are using this data don't have a defined logic to handle an undefined response.

Reply:
This is handled by isUserValid() from requests.js file